src/api_v2/models/user.rs

```diff
@@ -0,0 +1,73 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+use crate::api_v2::Error;
+
+/// ```
+/// if User::uid == 0{
+///     .. //bot user
+/// } else {
+///     .. //user
+/// };
+/// ```
+///
+pub struct User {
+    pub uid: i32,
+    pub time: u64,
```

Comment:
It will be better to use [`chrono::DateTime`](https://docs.rs/chrono/latest/chrono/struct.DateTime.html) here.

Reply:
unix timestamp can be converted,